4_Methodology/BS_SD.adoc

```diff
@@ -424,6 +427,9 @@ The evaluator shall perform the following test to verify that the TOE checks the
 The following test steps require the developer to provide access to a test platform that provides the evaluator with tools that are typically not found on factory products.
 
 . The evaluator shall present biometric samples of low quality for biometric verification that don’t satisfy the assessment criteria described in <<MBV assessment criteria for samples, assessment criteria for samples>>
++
+The evaluator may examine feedback messages (e.g. “Finger too wet” in case of fingerprint verification) shown when the TOE can’t capture the enough quality of samples to understand how the evaluator can present a low quality sample. The evaluator may also ask the developer how the low quality samples can be presented if the TOE doesn’t show any feedback messages.
```

Comment:
Same as above

```suggestion
. The evaluator may examine feedback messages shown (e.g. “Finger too wet” in case of fingerprint verification) to understand how the evaluator can present a biometric characteristic that does not meet the <<MBE assessment criteria for samples, assessment criteria for samples>>

. The evaluator may ask the developer how low quality samples can be created if the TOE doesn’t show any feedback messages
```